packages/extras/src/lib/hooks/useGltf.ts

```diff
@@ -10,12 +9,20 @@ import { buildSceneGraph, type SceneGraph } from '../lib/buildSceneGraph'
 import type { ThrelteGltf } from '../types/types'
 
 type UseGltfOptions = {
-  useDraco?: boolean | string
+  useDraco?: boolean | string | DRACOLoader
   useMeshopt?: boolean
   ktxTranscoderPath?: string
 }
 
-createEventDispatcher
+let defaultDracoLoaderInstance: DRACOLoader | undefined
```

Comment:
Make that a `Record<string, DRACOLoader>` (default: `{}`) where the key is the url of the loaders draco decoder path (for cross version caching), adapt the following function accordingly.